spark/src/main/scala/org/apache/mahout/drivers/RowSimilarityDriver.scala

```diff
@@ -135,7 +133,7 @@ object RowSimilarityDriver extends MahoutSparkDriver {
   }
 
   override def process: Unit = {
-    start()
+    start
```

Comment:
imo this is no small convention. at least i found it incredibly useful since they expand syntax of an attribute here that significant way. Note also attribute-esque nature of the convention since they also augmented 0-arity call with an assignemt "magic" syntax def name_=(value) which in code will just be an attribute assignment semantics such as `name = value`.

So this is not random.